npdarrington/travel-tracker
# Nd main body design

### What’s this PR do?  
- This PR is the initial basic structure of the HTML page with styling included.
- Utilizing SASS mixins to make a flexible flex display mixin.
- Placeholder for areas where JS can inject the user information after it is collected from the API.
 
### Where should the reviewer start?  
- Git pull this branch.
- Open `npm start` to launch the local server
- Run `npm install` if this has not been completed on local machine.
 
### How should this be manually tested?  
- After launching the local server, navigate to localhost:8080 in a browser.
- Ensure that a large background image, a form at the bottom of the background image, and the travel organizer section that displays a mock of a user's past trips, current trips, upcoming trips and pending trips displays at the bottom underneath the background image.
- Check SASS styling for any repeating styles.
 
### Any background context you want to provide?  
- No SASS variables have been utilized yet as color schemes have not been researched or finished yet.
- Utilized SASS styling using this great guide. Please refer to the guide an ensure that SASS styling is following the guide.
    - [SASS Style Guide - CSS Tricks](https://css-tricks.com/sass-style-guide/);
 
### What are the relevant tickets?  
- Partially addresses tickets #1, #3, #5 and #7. This tickets will be fully complete when the live data is injected using JS.
 
### Screenshots (if appropriate)  
<img width="1440" alt="Screen Shot 2020-09-17 at 6 55 48 PM" src="https://user-images.githubusercontent.com/59422609/93542965-9524c380-f917-11ea-898b-efc7df87aa24.png">
 
### Questions: 
- N/A
